qa/workunits/rbd/issue-20295.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -ex
+
+sudo ceph osd pool create ecpool 12 12 erasure
+sudo ceph osd pool application enable ecpool rbd
+sudo ceph osd pool set ecpool allow_ec_overwrites true
+sudo rbd --data-pool ecpool create --size 1024G test1
+sudo rbd bench-write test1 --io-pattern=rand
```

Comment:
We should probably be using `rbd bench --io-type write` instead of `bench-write`. The tool does say `bench-write` is deprecated in favor of the first.